cirq-core/cirq/devices/grid_device_metadata.py

```diff
@@ -39,12 +39,14 @@ def __init__(
         qubit_pairs: Iterable[Tuple['cirq.Qid', 'cirq.Qid']],
         gateset: 'cirq.Gateset',
         gate_durations: Optional[Dict['cirq.GateFamily', 'cirq.Duration']] = None,
+        isolated_qubits: Optional[Iterable['cirq.Qid']] = None,
```

Comment:
WDYT about using `all_qubits` instead?

Pros
* `DeviceSpecification` keeps a [full list of qubits](https://github.com/quantumlib/Cirq/blob/7c5ad613323c66d272959372f8f8b57d63fb1aa5/cirq-google/cirq_google/api/v2/device.proto#L22), so the translation is easier. Current design plans to keep this field. This will minimize changes on Pyle side.
* It will simplify the logic of computing the node set a bit, and also computing `all_qubits` for the super class constructor.

Cons
* If we want to expose an `isolated_qubits` property, still need to calculate that separately. This somewhat voids the benefit of reducing code for computing `all_qubits`.

Reply:
Just saw @mpharrigan 's comment :P

Reply:
Changed over to all_qubits approach. PTAL.